### DISCUSSION TITLE
Does wasmer support non standalone emscripten wasm?

### DISCUSSION THREAD

**Discussion by andersson09** [wasmerio/wasmer#2649] [Q&A] [answered]
Hey

Does wasmer support non standalone emscripten wasm which is bundled with the generated js file?

Thanks

**Comment by syrusakbary** [comment, marked as answer]
Hi @andersson09, we don't support the non-standalone emscripten files that require a JS engine at the moment (Nov 2021)

**Reply by andersson09** [reply]
Great thanks for getting back.
